openedx/core/djangoapps/credit/management/commands/delete_historical_credit_data.py

```diff
@@ -0,0 +1,33 @@
+"""
+Command to delete all rows from the credit_historicalcreditrequest and
+credit_historicalcreditrequirementstatus tables.
+"""
+
+import logging
+from django.core.management.base import BaseCommand
+from openedx.core.djangoapps.credit.models import CreditRequest, CreditRequirementStatus
+from openedx.core.djangoapps.util.row_delete import delete_rows
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
Shouldn't this be inheriting from BaseDeletionCommand instead?

Reply:
Yes indeed. Will fix.